src/api/client/Polymesh.ts

```diff
@@ -313,5 +316,12 @@ export class Polymesh {
   public get _middlewareApi(): ApolloClient<NormalizedCacheObject> {
     return this.context.middlewareApi;
   }
+
+  /**
+   * MiddlewareV2 client
+   */
+  public get _middlewareApiV2(): ApolloClient<NormalizedCacheObject> {
```

Comment:
This should get a test

Reply:
The others are marked as   /* istanbul ignore next: not part of the official public API */
Should I mark here as well?

Reply:
That works too.

I think a short one to add in `src/api/client/__tests__/Polymesh.ts` though.